diavlos/src/diaugeia/diaugeia.py

```diff
@@ -0,0 +1,56 @@
+"""A module for fetching diaugeia API information."""
+
+import logging
+import requests
+
+from .error import DiaugeiaErrorCode as ErrorCode
+
+logger = logging.getLogger(__name__)
+
+API_BASE_URL = "https://diavgeia.gov.gr/opendata"
+API_TEST_URL = "https://test3.diavgeia.gov.gr/luminapi/opendata/decisions/"
+
+
+class DiaugeiaError(Exception):
```

Comment:
This isn't actually raised anywhere. Remove it.

Reply:
TEST URL is used because production API does not return data. Needs research in diaugeia docs.

Reply:
Yeah, I know. My comment was about the `DiaugeiaError` class, not `API_TEST_URL`.